packages/cfd/src/Containers/__tests__/cfd-password-modal.spec.js

```diff
@@ -0,0 +1,513 @@
+        });
+    });
+
+    it('should display error message when password does not meet requirements', async () => {
+        validPassword.mockReturnValue(false);
+        const user_input = 'demo@deriv.com';
+        const props = {
+            ...mock_props,
+            is_cfd_password_modal_enabled: true,
+            is_cfd_success_dialog_enabled: false,
+            account_status: { status: [], category: 'Real' },
+            getAccountStatus: mockFn,
+            platform: 'mt5',
+            email: user_input,
+        };
+
+        render(
+            <Router history={history}>
+                <CFDPasswordModal {...props} />
+            </Router>
+        );
+        const ele_password_input = await screen.findByTestId('mt5_password');
+        fireEvent.change(ele_password_input, { target: { value: user_input } });
+        fireEvent.focusOut(ele_password_input);
+
+        await waitFor(() => {
+            expect(validPassword).toHaveBeenCalled();
+        });
+    });
+
+    it('should show password dialog asking user to provide POI', async () => {
+        const props = {
+            ...mock_props,
+            is_cfd_password_modal_enabled: true,
+            is_cfd_success_dialog_enabled: true,
+            is_password_modal_exited: true,
+            account_type: { category: 'real', type: 'financial_stp' },
+            getAccountStatus: mockFn,
+            platform: 'mt5',
+            is_eu: true,
+            is_fully_authenticated: false,
+        };
+
+        render(
+            <Router history={history}>
+                <CFDPasswordModal {...props} />
+            </Router>
+        );
+
+        expect(
+            await screen.findByText(/We need proof of your identity and address before you can start trading./i)
+        ).toBeInTheDocument();
+    });
+
+    it('should close the dialog when you click on Submit proof', async () => {
+        const mockDisableCFDPasswordModalFn = jest.fn();
+        const props = {
+            ...mock_props,
+            is_cfd_password_modal_enabled: true,
+            is_cfd_success_dialog_enabled: true,
+            is_password_modal_exited: true,
+            account_type: { category: 'real', type: 'financial_stp' },
+            getAccountStatus: mockFn,
+            platform: 'mt5',
+            is_eu: true,
+            is_fully_authenticated: false,
+            setCFDSuccessDialog: mockFn,
+            setMt5Error: mockDisableCFDPasswordModalFn,
+            disableCFDPasswordModal: mockFn,
+        };
+
+        render(
+            <Router history={history}>
+                <CFDPasswordModal {...props} />
+            </Router>
+        );
+
+        fireEvent.click(await screen.findByRole('button', { name: /Submit proof/i }));
+
+        await waitFor(() => {
+            expect(mockFn).toHaveBeenCalledWith(false);
+            expect(mockDisableCFDPasswordModalFn).toHaveBeenCalled();
+            expect(mockFn).toHaveBeenCalledWith(false);
+        });
```

Comment:
why do we have same assertion twice? line 299 and line 301?

Reply:
Will make the change